src/mnode/mnode-rpc.cpp

```diff
@@ -2333,6 +2333,79 @@ As json rpc
                 mnObj.pushKV("totalstoragefee", totalFee);
                 return mnObj;
             }
+            case RPC_CMD_LIST::validateownership: {
+
+                if (params.size() > 4)
+                {
+
+                    //result object
+                    UniValue retVal(UniValue::VOBJ);
+                    //txid
+                    std::string txid = params[2].get_str();
+                    //pastelid
+                    std::string pastelid = params[3].get_str();
+
+                    //Check if pastelid is found within the stored ones
+                    const auto pastelIDs = CPastelID::GetStoredPastelIDs();
+                    bool bIdFound = false;
+
+                    for (const auto & p: pastelIDs)
+                    {
+                        if(p.compare(pastelid) == 0)
+                        {
+                            bIdFound = true;
+                            break;
+                        }
+                    }
+
+                    if(bIdFound)
```

Comment:
it will be better to write throw here for visibility like
if (!bIdFound) {
  throw  ....
}

also then no need to shift next code at one level right

Reply:
Done